R/export.R

```diff
@@ -11,22 +11,26 @@ names(countries) <- unique(squire::population$iso3c)
 
 out_dir <- args[1]
 for (iso3c in names(countries)) {
+  R0 <- 3;
   pars <- parameters_explicit_SEEIR(
     countries[[iso3c]],
     hosp_bed_capacity = 1, #dummy value
-    ICU_bed_capacity = 1  #dummy value
+    ICU_bed_capacity = 1,  #dummy value
+    R0 = R0
   )
 
   country_data <- list(
     population = pars$population,
     contactMatrix = pars$mix_mat_set,
-    beta = pars$beta
+    beta = pars$beta,
+    eigen = R0 / pars$beta
```

Comment:
Would we ever want the eigenvectors too? If so consider renaming this to eigenvalue to be clearer